src/Corax/Queries/ITermProvider.cs

```diff
@@ -4,6 +4,5 @@ public interface ITermProvider
     { 
         void Reset();
         bool Next(out TermMatch term);
-        bool Evaluate(long id);
```

Comment:
We would need that when we implement query optimizer steps, why remove it?

Reply:
I am not entirely sure that is the proper way, and we are not using it yet. Removing it for now until we start to approach that.